### ISSUE TITLE
Redesign Survey and Table Top Appbar to meet ODK-X Design Guidelines 

### ISSUE THREAD

**Issue by Redeem-Grimm-Satoshi** [opendatakit/tool-suite-X#429]
**1**. Clone all 4 repositories locally into Android Studio ( Tables, Survey, Services and Android Library ) into the same folder, check  readme https://github.com/odk-x/survey and make sure they're running  without any errors

**Survey Top Appbar currently looks like this:**

![Screenshot_20231005_170535](https://github.com/odk-x/tool-suite-X/assets/45304978/a14c163c-a835-4867-95be-a574a5bcff01)

As you can see, It currently doesn't conform to ODK-X [Design Guidelines](https://docs.google.com/presentation/d/1Ol7YhG4WPLOWCoQZUrKHLPguHSiDz8RC3m8dNyTGGLo/edit#slide=id.ge7e6c91d25_0_5) 

**2.** Make changes to the Top Appbar so it conform to ODK-X Design Guidelines,

**3.** Send a Pull Request to Survey repository ( Development Branch ) with the changes and also a screenshot

###Read the [docs](docs.odk-x.org/) well so you know where to make required changes

**Comment by Chinex-Boroja** [comment]
@Redeem-Grimm-Satoshi can this task be assigned to me?

**Comment by cvemann** [comment]
Yes, please go ahead to work on it @Chinex-Boroja

**Comment by Chinex-Boroja** [comment]
> Yes, please go ahead to work on it @Chinex-Boroja

Alright. Thank you, I will update you on the progress

**Comment by Chinex-Boroja** [comment]
@Redeem-Grimm-Satoshi @wbrunette, I have set up my environment and cloned all four repositories. I am checking out the apps

**Comment by Chinex-Boroja** [comment]
Hi @Cveman1 @Redeem-Grimm-Satoshi, currently on this issue and I have a few things I am confused about while inspecting the code base.
- In the `res` folder 📁, checking the `values` sub-folder, I can't seem to locate the `styles.xml` where I can change the Top Appbar theme, also the same thing goes for the `colors.xml` file. I am a bit confused as to where to locate it, or am I to create them in the values folder?
- Secondly, this is how the Survey Top Appbar looks after running it on my android device, the blue app bar colour is not showing, which is a bit different from the screenshot sent
![Screenshot_2023-10-12-09-05-10-892_org opendatakit survey](https://github.com/odk-x/tool-suite-X/assets/61055200/3958d265-8edb-426e-ba4c-aef45d3d3643)

**Comment by Redeem-Grimm-Satoshi** [comment]
> Hi @Cveman1 @Redeem-Grimm-Satoshi, currently on this issue and I have a few things I am confused about while inspecting the code base.
> 
>     * In the `res` folder 📁, checking the `values` sub-folder, I can't seem to locate the `styles.xml` where I can change the Top Appbar theme, also the same thing goes for the `colors.xml` file. I am a bit confused as to where to locate it, or am I to create them in the values folder?
> 
>     * Secondly, this is how the Survey Top Appbar looks after running it on my android device, the blue app bar colour is not showing, which is a bit different from the screenshot sent
>       ![Screenshot_2023-10-12-09-05-10-892_org opendatakit survey](https://user-images.githubusercontent.com/61055200/274667329-3958d265-8edb-426e-ba4c-aef45d3d3643.jpg)

1. Great question, the app theming resides in the android library ( [read more here](https://developer.android.com/studio/projects/android-library) ) 
[
Android Library Repo](https://github.com/odk-x/androidlibrary) 

2. Yeah, Mine was altered don't worry about that

**Comment by Chinex-Boroja** [comment]
@Redeem-Grimm-Satoshi thank you. This provides a bit more context.

**Comment by Chinex-Boroja** [comment]
Hi @Redeem-Grimm-Satoshi @Cveman1 and @wbrunette , I am stuck on the Top Appbar color not reflecting. I have added the color values in the `color.xml` file, as described in the ODK-X Design. And also used them in the `theme.xml` file. But when running the app, the Appbar is not changing or showing the color changes. The color for the status bar is showing. This is the image screenshot
![Screenshot_2023-10-14-00-32-46-801_org opendatakit survey](https://github.com/odk-x/tool-suite-X/assets/61055200/628b6e23-643d-4aea-8d3a-f804b7283b2f)

But in Android Studio, I can see the colors properly reflected in the `sort.xml` file in the menu folder. This is also a screenshot to provide a bit more context

![App Bar shot](https://github.com/odk-x/tool-suite-X/assets/61055200/f73f5117-e4c4-4cc4-8ef5-f545addc9396)

I have also checked the Manifest file in the survey folder to investigate the app-theme. Probably I am missing something

**Comment by Redeem-Grimm-Satoshi** [comment]
Great, try to understand how appbars work : https://m2.material.io/components/app-bars-top/android#regular-top-app-bar

**Comment by Chinex-Boroja** [comment]
> Great, try to understand how appbars work : https://m2.material.io/components/app-bars-top/android#regular-top-app-bar

Thank you. I still have some question @Redeem-Grimm-Satoshi 
I figured out how to change the topbar app colour from `theme.xml` in the `androidlibrary module`. This is the update so far with a screenshot
![Screenshot_2023-10-20-18-44-24-313_org opendatakit survey](https://github.com/odk-x/tool-suite-X/assets/61055200/70023091-a351-463d-b812-b884d22f60f1)
- So is this what I am expected to do? Based on my understanding of the question.
- Secondly, it seems I am not touching any file on the `survey_app`, and I just want to find out if I am doing it the right way

I will appreciate a feedback, thank you

**Comment by Redeem-Grimm-Satoshi** [comment]
@Chinex-Boroja You did great

**Comment by Chinex-Boroja** [comment]
> @Chinex-Boroja You did great

Thank you @Redeem-Grimm-Satoshi, what should be my next step from here? Raising a Pull Request?

**Comment by Chinex-Boroja** [comment]
> > Great, try to understand how appbars work : https://m2.material.io/components/app-bars-top/android#regular-top-app-bar
> 
> Thank you. I still have some question @Redeem-Grimm-Satoshi I figured out how to change the topbar app colour from `theme.xml` in the `androidlibrary module`. This is the update so far with a screenshot ![Screenshot_2023-10-20-18-44-24-313_org opendatakit survey](https://user-images.githubusercontent.com/61055200/277016590-70023091-a351-463d-b812-b884d22f60f1.jpg)
> 
> * So is this what I am expected to do? Based on my understanding of the question.
> * Secondly, it seems I am not touching any file on the `survey_app`, and I just want to find out if I am doing it the right way
> 
> I will appreciate a feedback, thank you

Hi @Redeem-Grimm-Satoshi I am still expecting a response to the second question because it seems I did not touch any file from the `survey_app`. I only made changes to the `colors.xml` and `theme.xml` in the `androidLibrary`. So not sure on how to go about raising the pull request

**Comment by Redeem-Grimm-Satoshi** [comment]
> > > Great, try to understand how appbars work : https://m2.material.io/components/app-bars-top/android#regular-top-app-bar
> > 
> > 
> > Thank you. I still have some question @Redeem-Grimm-Satoshi I figured out how to change the topbar app colour from `theme.xml` in the `androidlibrary module`. This is the update so far with a screenshot ![Screenshot_2023-10-20-18-44-24-313_org opendatakit survey](https://user-images.githubusercontent.com/61055200/277016590-70023091-a351-463d-b812-b884d22f60f1.jpg)
> > 
> > * So is this what I am expected to do? Based on my understanding of the question.
> > * Secondly, it seems I am not touching any file on the `survey_app`, and I just want to find out if I am doing it the right way
> > 
> > I will appreciate a feedback, thank you
> 
> Hi @Redeem-Grimm-Satoshi I am still expecting a response to the second question because it seems I did not touch any file from the `survey_app`. I only made changes to the `colors.xml` and `theme.xml` in the `androidLibrary`. So not sure on how to go about raising the pull request

Yes you didn't because Android Library is where other ODK-X apps pull their resources from ( eg theming, etc.. )... 

A PR can be sent to the Android Library Repo

**Comment by Chinex-Boroja** [comment]
Thanks for the clarification @Redeem-Grimm-Satoshi

**Comment by Chinex-Boroja** [comment]
@Redeem-Grimm-Satoshi I just raised a [PR ](https://github.com/odk-x/androidlibrary/pull/263)on this issue
